src/main/java/org/gridsuite/loadflow/server/LoadFlowController.java

```diff
@@ -41,19 +45,22 @@ public class LoadFlowController {
     @Autowired
     private LoadFlowService loadFlowService;
 
+    @Autowired
+    private LoadFlowWorkerService loadFlowWorkerService;
+
     @PutMapping(value = "/networks/{networkUuid}/run", produces = APPLICATION_JSON_VALUE)
     @Operation(summary = "Run a load flow on a network")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The load flow has been performed")})
-    public ResponseEntity<LoadFlowResult> run(@Parameter(description = "Network UUID") @PathVariable("networkUuid") UUID networkUuid,
+    public ResponseEntity<Mono<LoadFlowResult>> run(@Parameter(description = "Network UUID") @PathVariable("networkUuid") UUID networkUuid,
```

Comment:
From a REST API point of view, this is still blocking. The purpose of this PR was to have the same design as for instance security analysis, with a run that do not return anything and that is completing immediately and some other methods to get status and results.

Reply:
Also we should go to a rabbitmq queue between API service and worker service to manage the asynchrone calculation

Reply:
And probably manage the running state and result through DB